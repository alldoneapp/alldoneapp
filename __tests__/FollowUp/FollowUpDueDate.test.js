import React from 'react'
import FollowUpDueDate from '../../components/FollowUp/FollowUpDueDate'
import renderer from 'react-test-renderer'

describe('FollowUpDueDate component', () => {
    describe('FollowUpDueDate snapshot test', () => {
        it('should render correctly', () => {
            const tree = renderer
                .create(
                    <FollowUpDueDate
                        closePopover={() => {}}
                        selectDate={true}
                        onCustomDatePress={() => {}}
                        dateText="Today"
                    />
                )
                .toJSON()
            expect(tree).toMatchSnapshot()
        })

        it('should render correctly', () => {
            const tree = renderer
                .create(
                    <FollowUpDueDate
                        closePopover={() => {}}
                        selectDate={false}
                        onCustomDatePress={() => {}}
                        dateText="Today"
                    />
                )
                .toJSON()
            expect(tree).toMatchSnapshot()
        })
    })
})
