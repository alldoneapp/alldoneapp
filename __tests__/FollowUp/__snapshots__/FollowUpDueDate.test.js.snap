// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FollowUpDueDate component FollowUpDueDate snapshot test should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#091540",
      "borderRadius": 4,
      "elevation": 3,
      "shadowColor": "rgba(78, 93, 120, 0.56)",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 16,
      "width": 305,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingLeft": 16,
        "paddingRight": 8,
        "paddingTop": 8,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "column",
          "marginTop": 8,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#04142F",
              "fontFamily": "Roboto-Medium",
              "fontSize": 18,
              "lineHeight": 26,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Select Follow up reminder
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#04142F",
              "fontFamily": "Roboto-Regular",
              "fontSize": 14,
              "letterSpacing": 0.32,
              "lineHeight": 22,
            },
            Object {
              "color": "#8A94A6",
              "width": 273,
            },
          ]
        }
      >
        Select from the options below
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingBottom": 8,
        "paddingHorizontal": 16,
        "paddingTop": 20,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Today
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "fontFamily": "Roboto-Medium",
                "fontSize": 16,
                "letterSpacing": 0.16,
                "lineHeight": 24,
              },
              Object {
                "color": "white",
              },
            ]
          }
        >
          22 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Tomorrow
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          23 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        This next Saturday
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          04 May
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        This next Monday
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          29 Apr
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "rgba(255,255,255,0.2)",
        "borderBottomWidth": 1,
        "borderTopColor": "rgba(255,255,255,0.2)",
        "borderTopWidth": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 2 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          24 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 4 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          26 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 7 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          29 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 30 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          22 May
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "rgba(255,255,255,0.2)",
        "borderBottomWidth": 1,
        "borderTopColor": "rgba(255,255,255,0.2)",
        "borderTopWidth": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Pick a date
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          <View>
            <Text
              style={
                Object {
                  "color": "#8A94A6",
                  "fontFamily": "alldone",
                  "fontSize": 24,
                  "opacity": 1,
                }
              }
            >
              
            </Text>
          </View>
        </Text>
      </View>
    </View>
  </View>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
    style={
      Array [
        Object {
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <View
      accessible={true}
      style={
        Object {
          "flexDirection": "row",
          "opacity": 1,
          "paddingLeft": 16,
          "paddingVertical": 16,
        }
      }
    >
      <View>
        <Text
          style={
            Object {
              "color": "#8A94A6",
              "fontFamily": "alldone",
              "fontSize": 24,
              "opacity": 1,
            }
          }
        >
          
        </Text>
      </View>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "#FFFFFF",
              "fontWeight": "500",
              "marginLeft": 8,
            },
          ]
        }
      >
        Back
      </Text>
    </View>
  </RNGestureHandlerButton>
  <View
    style={
      Object {
        "position": "absolute",
        "right": 13,
        "top": 13,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View>
        <Text
          style={
            Object {
              "color": "#8A94A6",
              "fontFamily": "alldone",
              "fontSize": 24,
              "opacity": 1,
            }
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`FollowUpDueDate component FollowUpDueDate snapshot test should render correctly 2`] = `
<View
  style={
    Object {
      "backgroundColor": "#091540",
      "borderRadius": 4,
      "elevation": 3,
      "shadowColor": "rgba(78, 93, 120, 0.56)",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 16,
      "width": 305,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingLeft": 16,
        "paddingRight": 8,
        "paddingTop": 8,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "column",
          "marginTop": 8,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#04142F",
              "fontFamily": "Roboto-Medium",
              "fontSize": 18,
              "lineHeight": 26,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Select Follow up reminder
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#04142F",
              "fontFamily": "Roboto-Regular",
              "fontSize": 14,
              "letterSpacing": 0.32,
              "lineHeight": 22,
            },
            Object {
              "color": "#8A94A6",
              "width": 273,
            },
          ]
        }
      >
        Select from the options below
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingBottom": 8,
        "paddingHorizontal": 16,
        "paddingTop": 20,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Today
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "fontFamily": "Roboto-Medium",
                "fontSize": 16,
                "letterSpacing": 0.16,
                "lineHeight": 24,
              },
              Object {
                "color": "white",
              },
            ]
          }
        >
          22 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Tomorrow
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          23 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        This next Saturday
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          04 May
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        This next Monday
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          29 Apr
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "rgba(255,255,255,0.2)",
        "borderBottomWidth": 1,
        "borderTopColor": "rgba(255,255,255,0.2)",
        "borderTopWidth": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 2 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          24 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 4 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          26 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 7 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          29 Apr
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        In 30 days
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          22 May
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "rgba(255,255,255,0.2)",
        "borderBottomWidth": 1,
        "borderTopColor": "rgba(255,255,255,0.2)",
        "borderTopWidth": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "white",
            },
          ]
        }
      >
        Pick a date
      </Text>
      <View
        style={
          Object {
            "marginLeft": "auto",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#04142F",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "letterSpacing": 0.32,
                "lineHeight": 24,
              },
              Object {
                "color": "#8A94A6",
              },
            ]
          }
        >
          <View>
            <Text
              style={
                Object {
                  "color": "#8A94A6",
                  "fontFamily": "alldone",
                  "fontSize": 24,
                  "opacity": 1,
                }
              }
            >
              
            </Text>
          </View>
        </Text>
      </View>
    </View>
  </View>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
    style={
      Array [
        Object {
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <View
      accessible={true}
      style={
        Object {
          "flexDirection": "row",
          "opacity": 1,
          "paddingLeft": 16,
          "paddingVertical": 16,
        }
      }
    >
      <View>
        <Text
          style={
            Object {
              "color": "#8A94A6",
              "fontFamily": "alldone",
              "fontSize": 24,
              "opacity": 1,
            }
          }
        >
          
        </Text>
      </View>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Medium",
              "fontSize": 16,
              "letterSpacing": 0.16,
              "lineHeight": 24,
            },
            Object {
              "color": "#FFFFFF",
              "fontWeight": "500",
              "marginLeft": 8,
            },
          ]
        }
      >
        Back
      </Text>
    </View>
  </RNGestureHandlerButton>
  <View
    style={
      Object {
        "position": "absolute",
        "right": 13,
        "top": 13,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View>
        <Text
          style={
            Object {
              "color": "#8A94A6",
              "fontFamily": "alldone",
              "fontSize": 24,
              "opacity": 1,
            }
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;
